backend/routes/user.js

```diff
@@ -21,7 +22,6 @@ routes.get('/:userid', async (req, res, next) => {
     user = await getUser(userId, requestingUserId);
   } catch (e) {
     next(e);
-    return;
```

Comment:
Why are we removing the returns after catching errors? I don't think this adds any functionality, and we don't want to slip out of the catch case and return an incorrect response